### PR TITLE
SegmentedControl comonent selected segment  text bug in android

### PR DESCRIPTION
**Pull request by MingSeng-W**
# Summary
<p>&nbsp;&nbsp;The SegmentedControl component has a bug in Android because the selected segment text color is forced to be written as "#fff". When tintColor is set to "#fff", the selected segment text is invisible. The modified code can be used to set the current active text color via the style attribute. If not set, the default is "#fff". 
</p>
<p>&nbsp;&nbsp;SegmentedControl组件在安卓的情况下出现bug，原因是被选中的segment text颜色被强制写死成"#fff",在tintColor被设置为"#fff"的时候，被选中的segment文本看不见。修改过后的代码可以通过style属性来设置当前活跃文本颜色，如果没有设置的话默认为"#fff"。</p>

# Steps To Reproduce
<p>ant-design-mobile-rn: 2.3.3</p>
<p>device: android phone</p>
<p>
   import the SegmentedControl from ant-design-mobile-rn, and set the tintColor to “#fff”
</p>

pr

# Changelog
<p>[Android] [Fixed] fix the selected segment bug in android</p>
<h1>Test Plan</h1>

- [ done] Edit the demo of this componet,  set the "selectedSegTextColor" props to 'black',  check the commit does take effects in android simulator
```html
        <Text>TintColor and Style</Text>
        <SegmentedControl
          values={['Segment1', 'Segment2', 'Segment3']}
          tintColor={'#ff0000'}
          style={{ height: 40, width: 280 }}
          selectedSegTextColor={'black'}
        />
```
image

- [done]   npm run test -- components/segmented-control/__tests__/index.test.js ,  the commit pass the test
image

### CODE REVIEW

**Comment by bang88** [comment]
Thank you for your contribution. Please fixes the CI error 😃

**Comment by MingSeng-W** [comment]
> Thank you for your contribution. Please fixes the CI error 😃

ts小白,可以帮忙看下lint的error么，google试了好多方法都无效，折磨我好久了%>_<%。

File: components/segmented-control/segmented.android.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ export default class SegmentedControl extends React.Component<
                   // tslint:disable-next-line:jsx-no-multiline-js
                   style={[
                     styles.itemText,
-                    { color: idx === selectedIndex ? style.color == undefined ? '#fff' : style.color : tintColor },
+                    { color: idx === selectedIndex ? style.color === undefined ? '#fff' : style.color : tintColor },
```
Comment (bang88) [inline components/segmented-control/segmented.android.tsx]: 这里style有可能是undefined，所以要检查一下style，然后还要看看 style 里面有没有这个color的属性。

Comment (MingSeng-W) [reply components/segmented-control/segmented.android.tsx]: thanks,  是否可以给这个组件新增个属性比如说selectedSegTextColor, 或者通过在配置theme的方式，来给当前selected的段设置color,  style.color用起来容易造成歧义。

Comment (bang88) [reply components/segmented-control/segmented.android.tsx]: You can do it.

File: components/segmented-control/PropsType.tsx [hunk 2]
```diff
@@ -5,5 +5,5 @@ export interface SegmentedControlPropsType {
   values?: string[];
   onChange?: (e: any) => void;
   onValueChange?: (value: string) => void;
-  selectedSegTextColor?: string  //for android platform
+  selectedSegTextColor?: string;  //for android platform
```
Comment (bang88) [inline components/segmented-control/PropsType.tsx]: Please rename to `selectedTextColor` . 😄

**Comment by bang88** [comment]
@MingSeng-W Thank you. almost done here. Please consider naming it as simple as.

**Comment by MingSeng-W** [comment]
> @MingSeng-W Thank you. almost done here. Please consider naming it as simple as.

done

**Comment by bang88** [comment]
有空的话 顺道看一下 #474 #511  😄

**Comment by MingSeng-W** [comment]
👌🏼

**Pull request merged**